### ISSUE TITLE
any plan to support mexc?

### ISSUE THREAD

**Issue by yingbiu** [thrasher-corp/gocryptotrader#1139]
# New Issue

## Context
 mexc is top 17th exchange ,any plan to support it?

**Comment by thrasher-** [comment]
Yep we do, just once we clear out some of the other exchanges we're working on!
